style.css

```
@@ -0,0 +1,346 @@
+body {
+    background-color: #f1f1f1;
+    font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+    font-size: 14px;
+    color: #505050;
+    line-height: 16.41px;
+    word-wrap: break-word;
+}
+
+.container, .container-fluid {
+    padding: 0;
+}
+
+.wrapper {
+    margin-top: 136px;
+}
+
+header {
+    background-color: #ffffff;
+    left: 0;
+    top: 0;
+    width: 100%;
+}
+
+main {
+    object-fit: cover;
+}
+
+#header_main_nav, #header_nav {
+    box-shadow: 0 1px 1px 0 #0000001A;
+}
+
+#header_nav {
+    background-color: #fafafa;
+}
+
+#header_nav li {
+    margin: 0;
+}
+
+header nav {
+    height: 40px;
+}
+
+h1 {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 21.09px;
+}
+
+.h_span {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 21.09px;
+}
+
+.viewers_count, .comments_count {
+    line-height: 18.75px;
+}
+
+.nav-link {
+    color: #505050;
+    text-decoration: none;
+}
+
+.nav-link:focus, .nav-link:hover {
+    color: #505050;
+}
+
+p {
+    margin: 0;
+}
+
+h4 {
+    font-size: 14px;
+    font-weight: 500;
+    line-height: 20px;
+    margin: 0;
+}
+
+#main_nav div {
+    height: 48px;
+    width: 60px;
+}
+
+#navigation {
+    max-height: 312px;
+}
+
+header nav ul {
+    height: 100%;
+}
+
+header nav li { 
+    align-items: center;
+    display: flex;
+    height: 100%;
+    justify-content: center;
+}
+
+#map_img {
+    height: 400px;
+}
+
+.nav-link.active, .nav-link.active {
+    color: #505050 !important;
+    background-color: #f1f1f1 !important;
+    border: none !important;
+} 
+
+.nav-tabs, .nav-link {
+    margin-bottom: 0 !important;
+    border: none !important;
+    border-top-left-radius: 0 !important; 
+    border-top-right-radius: 0 !important;
+    border-bottom: none !important;
+    height: 100%;
+}

```

!important это самое крайнее что можно сделать, ты можешь увеличить немного селектор, или использовать такой же как и в бутстрап, и твои стили будут приоритетнее, тк грузятся ниже в html, ну и много одинаковых стилей

```suggestion
.nav-tabs,
.nav-tabs .nav-link,
.nav-tabs .nav-link.active {
    margin-bottom: 0;
    border: none;
    border-radius: 0; 
    height: 100%;
}

.nav-tabs .nav-link.active {
    color: #505050;
    background-color: #f1f1f1;
} 
```